src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -25,26 +27,43 @@
 
 
 class TestMLBF(TestCase):
+    def setUp(self):
+        self.user = user_factory()
+
+    def _blocked_addon(self, **kwargs):
```

Comment:
This might seem odd, but in the next PR I will almost totally replace the existing tests and this is how I generate the data, so easier to just add it now as there are a few tests that were added in this PR. https://github.com/mozilla/addons-server/pull/22663